tests/scaffolders/plaso_sqlite.py

```diff
@@ -0,0 +1,65 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Tests for the plaso sqlite scaffolder."""
+import unittest
+
+from plasoscaffolder.lib import errors
+from plasoscaffolder.scaffolders import plaso_sqlite
+
+
+class PlasoSqliteScaffolderTest(unittest.TestCase):
```

Comment:
SQLite - there's a bunch of other typos of SQLite in this file too - please fix

Reply:
done